web_development_101/installations/setting_up_git.md

```diff
@@ -121,6 +121,26 @@ Now, go back to GitHub in your browser window and paste the key you copied into
 
 #### Step 2.5 Testing your key
 
+First, verify that your new key is being used:
+
+~~~bash
+ssh-add -l -E md5
+~~~
+
+The output should be:
+
+~~~bash
+2048 MD5:[the signature] [your email] (RSA)
+~~~
+
+If the output is "The agent has no identities.", you need to add it. For Linux/WSL, enter:
+
+~~~bash
+echo 'ssh-add ~/.ssh/id_rsa &>/dev/null' >> ~/.bashrc
+~~~
+
+If you are on MacOS, simply replace "bashrc" with "bash_profile".
```

Comment:
This will have people asking where "bashrc" is

Reply:
True, I clearly don't have the skills to write an efficient lesson. I mainly did this to raise the issue and there definitely are better ways to formulate this.